lib/bamboo/plug/helper.ex

```diff
@@ -17,6 +17,12 @@ defmodule Bamboo.EmailPreviewPlug.Helper do
     |> Enum.join(", ")
   end
 
+  def format_values(values) when is_binary(values), do: values
+  def format_values(values) when is_list(values) do
+    Enum.join(values, ", ")
+  end
+  def format_values(values), do: inspect(values)
```

Comment:
Also, what is this catch all function for? I think headers will always be a list or a string, or is there something else that it could be?